### DISCUSSION TITLE
Import and initialize prisma

### DISCUSSION THREAD

**Discussion by Juses24** [prisma/prisma#20828] [Q&A]
### Bug description

@prisma/client did not initialize yet

### Prisma information

```prisma
// Add your schema.prisma
```

```ts
// Add your code using Prisma Client
```

### Environment & setup

- OS: 
- Database: 
- Node.js version: 

### Prisma Version

```

```

**Comment by Juses24** [comment]
i just import prisma, but he cannot initialize

**Reply by ludralph** [reply]
Hi @Jesusdasilva2001 👋 

Can you please run the command below to generate the Prisma Client?
```
npx prisma generate
```

If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏

**Comment by Juses24** [comment]
Hey @ludralph , thanks for your time. I finish the project, i have to write wrong the words, i run npm hhhhhhhh , i´ts crazy, but i did it.
